cobalt/build/configs/common.gn

```diff
@@ -15,3 +15,5 @@ ffmpeg_branding = "Chrome"
 # Removing these flags causes a lot of compiler errors in starboard code.
 clang_use_chrome_plugins = false
 enable_check_raw_ptr_fields = false
+
+optimize_for_size = true
```

Comment:
what's the default ? also just from the name doesn't seem like we want this for debug/devel builds - might affect how much debug information is present in the binary/ debug logging etc

Reply:
https://chromium.googlesource.com/chromium/src/+/d1a04b008747abf9231b676ad1b34e53bad43973/build/config/compiler/BUILD.gn#74
It is defined here, but I do not know if it exists in our build system.

Reply:
oh, great, thanks.